plugins/API/ProcessedReport.php

```diff
@@ -363,9 +363,13 @@ public function getProcessedReport($idSite, $period, $date, $apiModule, $apiActi
 
         list($newReport, $columns, $rowsMetadata, $totals) = $this->handleTableReport($idSite, $dataTable, $reportMetadata, $showRawMetrics, $formatMetrics);
 
-        foreach ($columns as &$name) {
-            $name = ucfirst($name);
-        }
+	if (function_exists('mb_substr)) {
+            foreach ($columns as &$name) {
+                if (substr($name, 0, 1) === mb_substr($name, 0, 1) {
```

Comment:
Well, maybe the best solution is add an option to translate file to idencate if this language can have upper case.

But for now, mb_substr should solve most the issue (for me). and prevent lot's of crash (if using opcache)